apps/xprof_gui/priv/src/actions/NotificationActions.js

```diff
@@ -43,8 +44,10 @@ export const addNotification = (
   const lastId = getNotificationsLastId(state);
   const notification = {
     id: lastId + 1,
-    severity,
-    message,
+    severity: (error && error.data && error.data.severity)
```

Comment:
is there a nicer way to check whether a deep object field is defined?

Reply:
you can use lodash -` _.get(error, 'error.data.severity'))`, however it isn't nicer :P

Reply:
😄  actually with a default value its more compact and less redundant, I like it
```
_.get(error, 'data.severity', defaultSeverity)
```